<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scanlabelsClass</class>
 <widget class="QWidget" name="scanlabelsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ανάγνωση Ετικετών</string>
  </property>
  <widget class="QLineEdit" name="scanlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="scanLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="scantableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>371</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="contpushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Συνέχεια</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
